frontend/src/components/maps/leaflet/mapUtils.tsx

```diff
@@ -161,7 +161,7 @@ export const toCqlFilter = (object: any, pidOverride?: boolean) => {
   const cql: string[] = [];
   Object.keys(object).forEach((key: string) => {
     if (object[key]) {
-      if (key === 'PID' && object[key] && !pidOverride) {
+      if (key === 'PID' || (key === 'PID_PADDED' && object[key] && !pidOverride)) {
```

Comment:
Question - is this change related to this PR? I saw another PR with this change already...